tests/test_router_baseline.py

```
@@ -0,0 +1,303 @@
+        random.Random(3).shuffle(shuffled)
+        second = rb.split(shuffled)
+        self.assertEqual(
+            sorted(s.record_id for s in first[1]),
+            sorted(s.record_id for s in second[1]),
+        )
+        twin = rb.Sample(
+            record_id="twin-of-first",
+            features=samples[0].features,
+            label=samples[0].label,
+        )
+        train, test = rb.split(samples + [twin])
```

Ruff (0.16.3)

[warning] Unpacked variable `train` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)